packages/path_provider/path_provider_linux/test/path_provider_linux_test.dart

```diff
@@ -35,8 +36,19 @@ void main() {
   });
 
   test('getApplicationSupportPath', () async {
-    final PathProviderPlatform plugin = PathProviderPlatform.instance;
-    expect(await plugin.getApplicationSupportPath(), startsWith('/'));
+    final PathProviderPlatform plugin = PathProviderLinux.private(
+        executableName: 'test', applicationId: 'com.example.Test');
+    // Note this will fail if ${xdg.dataHome.path}/test exists on the local filesystem.
```

Comment:
Maybe use something less likely to collide? E.g., `path_provider_linux_unit_test`